var/spack/repos/builtin/packages/maphyspp/package.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Maphyspp(CMakePackage):
+    """a Massively Parallel Hybrid Solver in C++"""
+
+    homepage = "https://gitlab.inria.fr/solverstack/maphys/maphyspp"
+    git = "https://gitlab.inria.fr/solverstack/maphys/maphyspp.git"
+    url =  "https://gitlab.inria.fr/api/v4/projects/6194/packages/generic/source/v1.1.9/maphyspp-1.1.9.tar.gz"
+    maintainers("fpruvost")
+
+    version('1.1.9', '472deef05f69c26337a6f8e769cf36cbe0a50e6ec096757389ed10286a0d7e04',
+            url='https://gitlab.inria.fr/api/v4/projects/6194/packages/generic/source/v1.1.9/maphyspp-1.1.9.tar.gz')
+    version("master", branch="master",  submodules=True)
+
+    # User options
+    variant("armadillo", default=False, description="Compile examples and tests using Armadillo")
+    variant("pastix", default=True,  description="Compile examples and tests using pastix (sparse direct solver)")
+    #variant("mumps", default=False, description="Compile examples and tests using mumps (sparse direct solver)") #FIXME
```

Comment:
Do you plan to fix these commented out variants before or after the merge?  (Style checks will likely complain about the lack of a space after the comment marks.)